htdocs/comm/remx.php

```diff
@@ -378,7 +378,7 @@
 			// VAT
 			print '<tr><td>'.$langs->trans("VAT").'</td>';
 			print '<td>';
-			print $form->load_tva('tva_tx', GETPOSTISSET('tva_tx') ? GETPOST('tva_tx', 'alpha') : 0, $mysoc, $object, 0, 0, '', 0, 1);
+			print $form->load_tva('tva_tx', (GETPOSTISSET('tva_tx') ? GETPOST('tva_tx', 'alpha') : (!empty(getDolGlobalString('MAIN_VAT_DEFAULT_IF_AUTODETECT_FAILS')) ? getDolGlobalString('MAIN_VAT_DEFAULT_IF_AUTODETECT_FAILS') : 0)), $mysoc, $object, 0, 0, '', 0, 1);
```

Comment:
Do not use !empty() on getDolGlobalString(), it has becale useless.
You can replace:
(!empty(....  ...0))
With just
getDolGlobalString('MAIN_VAT_DEFAULT_IF_AUTODETECT_FAILS', '0')